packages/core/src/app/common/error/ErrorModal.spec.tsx

```diff
@@ -11,6 +11,7 @@ import { Modal } from '../../ui/modal';
 
 import ErrorCode from './ErrorCode';
 import ErrorModal, { ErrorModalProps } from './ErrorModal';
+import { CustomError } from './index';
```

Comment:
This seems like a small enough test case, can you please convert this to RTL?

Reply:
Also you can update the file format from `.spec.tsx` to `.test.tsx`